pom.xml

```diff
@@ -47,89 +62,424 @@
 		</developer>
 	</developers>
 
-	<pluginRepositories>
-		<pluginRepository>
-			<id>repo.jenkins-ci.org</id>
-			<url>http://repo.jenkins-ci.org/public/</url>
-		</pluginRepository>
-	</pluginRepositories>
+	<properties>
+		<!-- Plugin dependencies -->
+		<jenkinsCoreVersion>1.651.1</jenkinsCoreVersion>
+		<gitPluginVersion>2.4.4</gitPluginVersion>
+		<jqueryPluginVersion>1.11.2-0</jqueryPluginVersion>
+		<jodaTimeVersion>2.9.2</jodaTimeVersion>
+		<commonsLang3Version>3.4</commonsLang3Version>
+		<!-- Jenkins dependencies -->
```

Comment:
I don't agree, to add "all" jenkins dependency to pom file. 
Only necessary dependency.